front/lib/api/assistant/configuration.ts

```diff
@@ -535,6 +543,18 @@ async function fetchWorkspaceAgentConfigurationsForView(
         });
       }
 
+      const websearchConfigurations = websearchConfigs[agent.id] ?? [];
```

Comment:
Is it time to refactor into a generic "for all action types" rather? (to avoid the same omission that led to this fix)

Can do it directly here or in a follow up PR.
WDYT?

Reply:
This should be refactored indeed to use the new Runner classes :+1: 

Let's add websearch here and refactor the whole thing in a follow-up PR :+1: